src/client/css/partials/cancelSubscriptionSurvey.css

```diff
@@ -0,0 +1,37 @@
+[data-partial='cancelSubscriptionSurvey'] {
+  text-align: center;
+}
+
+[data-partial='cancelSubscriptionSurvey'] header, form {
```

Comment:
Q: Can you confirm this is correct, or does the `form` need a `[data-partial=...]` prefix as well?
I'm unclear on this will apply to ALL forms or if this .css file is only loaded on one page and it's low risk.

Reply:
ah, yeah, I think the comma above _might_ make a difference?
Here's the version from addEmail.css:
```css
[data-partial='addEmail'] form {
  display: flex;
  flex-direction: column;
  align-items: center;
  gap: var(--padding-lg);
}
```

Reply:
Yeah, I think CSS nesting would help greatly here, but doesn't look like we have great native browser support yet: https://caniuse.com/css-nesting